settingup.tex

```diff
@@ -101,25 +100,25 @@ \subsection{Installing the Real-Time Clock Battery}
 
 The MEGA65 includes a Real-Time Clock, which is used to display the time and date on the startup screen, to add timestamps to files that the MEGA65 writes to your SD cards, and to provide the {\bf DT\$}\index{BASIC 65 Variables!DT} and {\bf TI\$}\index{BASIC 65 Variables!TI\$} BASIC variables for use in programs. This clock uses a CR2032 coin-cell battery to keep the time when the MEGA65 is disconnected from power for long periods of time. The MEGA65 does not include a battery in order to avoid issues related to shipping batteries internationally.\footnote{Early models of MEGA65 with the ``R3A'' board revision (made in the year 2022) use a battery of type CR1220 for the Real-Time Clock, and the battery is required for the RTC to function. Revision ``R6'' (made in 2024) uses a battery of type CR2032, and the battery is optional for regular use.}
 
-{\bf The RTC battery is optional.} The MEGA65 can keep the RTC running without a battery, even when the computer is disconnected from power, for multiple days at a time. Installing a battery allows the computer to remember the time for much longer periods. (You can always set the clock again later.)
+{\bf The RTC battery is optional.} The MEGA65 can keep the RTC running without a battery, even when the computer is disconnected from power, for multiple days at a time. Installing a battery allows the computer to remember the time for much longer periods. You can always set the clock again later.
 
 To install the battery, switch off the computer. Use a Phillips-head screwdriver to open the case, exposing the motherboard.\index{Case!Opening} The case is held together with three screws, all of which are along the bottom of the front side of the case. Once the screws have been removed, carefully lift the top half of the case. Note the orientation of the keyboard connector, then disconnect it.
 
 The battery is located between the controller ports and the keyboard connector.
 
 \includegraphics[width=\linewidth]{images/illustrations/rtc-battery-location.pdf}
 
-If you are removing an existing battery, push the battery release lever on the bottom (flat-sided) side of the battery socket away from the battery to remove it. Insert the new battery with the side labelled {\bf +} facing up, and press it into place.
+If you are removing an existing battery, push the battery release lever on the bottom (flat-sided) side of the battery socket away from the battery to remove it. Insert the new battery with the positive side (labelled {\bf +}) facing up, and press it into place.
 
 Once you have re-assembled your MEGA65, you can set the time in the Configuration Utility. For more information on how to set the Real-Time Clock, refer to the Configuration Utility section on page \pageref{sec:configuration-utility}.
 
 
 \newpage
 
-\section{Switching the MEGA65 on for the First Time}
+\section{Powering up for the First Time}
```

Comment:
See previous comment about "switching on." We try not to use synonyms for "switching on the computer."

Reply:
(I have no personal preference here and accept the power/switch rationale you guys have. )

My main concern here is to avoid chapter names that break lines into two. 😉

Would something shorter, such as "Switching on for the first time" or "Switching on the MEGA65", whichever will not break into two lines, work?

Reply:
I agree with the goal. "Switching on the MEGA65" sounds fine to me.

While you're in here can you change the title of this chapter to be "Setting Up"? I just noticed it says "Setup." :)

Reply:
Done!